spectator/registry.go

```diff
@@ -0,0 +1,214 @@
+// Package spectator provides a minimal Go implementation of the Netflix Java
+// Spectator library. The goal of this package is to allow Go programs to emit
+// metrics to Atlas.
+//
+// Please refer to the Java Spectator documentation for information on
+// spectator / Atlas fundamentals: https://netflix.github.io/spectator/en/latest/
+package spectator
+
+import (
+	"encoding/json"
+	"github.com/Netflix/spectator-go/spectator/meter"
+	"github.com/Netflix/spectator-go/spectator/writer"
+	"io/ioutil"
+	"path/filepath"
+)
+
+// Config represents the Registry's configuration.
+type Config struct {
+	CommonTags map[string]string `json:"common_tags"`
```

Comment:
Spectatord becomes responsible for almost all common tags.

These common tags should only be applying values from `NETFLIX_PROCESS_NAME` and `TITUS_CONTAINER_NAME`.

https://github.com/Netflix/spectator-py/blob/main/spectator/commontags.py#L17-L23

Reply:
I'll add those two tags. 

I don't know if I'm reading this correctly but it seems like [spectator-py impl](https://github.com/Netflix/spectator-py/blob/main/spectator/sidecarconfig.py#L34) ignores those two tags if a value is set in the config, whereas Java impl would merge the tags. I think I prefer merging.

Reply:
Hm, I had forgotten about that. The idea there was to allow people to completely override some or all of the Netflix common tags, if they provide a config this way. SpectatorD allows people to override common tags by providing them on the command line.

Setting common tags here may or may not clash with what SpectatorD is doing. In a workstation scenario, spectatord would be running without a Netflix environment and thus have no or very few common tags - they need to all be supplied by the user. I don't remember the exact use case here.

I think merging is fine - that supports a more common use case of adding XYZ tag to every metric, even if it is not of the `nf.* ` variety.

Reply:
Sounds good, that's what I ended up implementing